Frameworks/CoreData/NSManagedObjectModel.mm

```diff
@@ -14,17 +14,106 @@
 //
 //******************************************************************************
 
+#import <ErrorHandling.h>
 #import <StubReturn.h>
-#import <CoreData/NSManagedObjectModel.h>
+
+#import <CoreData/CoreData.h>
+#import <Foundation/Foundation.h>
+
+#import "NSManagedObjectModel-XMLParsing.h"
+
+@interface NSManagedObjectModel () {
+    StrongId<NSMutableArray> _unresolvedEntities;
+}
+@property (atomic, copy) NSDictionary<NSString*, NSEntityDescription*>* entitiesByName;
+@end
 
 @implementation NSManagedObjectModel
 /**
-@Status Stub
-@Notes
+ @Status Caveat
+ @Notes Parses all relationships, entities, attributes and ignopres configurations and fetch requests.
 */
-- (instancetype)initWithContentsOfURL:(NSURL*)url {
-    UNIMPLEMENTED();
-    return StubReturn();
+- (id)initWithContentsOfURL:(NSURL*)url {
+    [self release];
```

Comment:
release when we enter this function? or did we intend this be released when url is nil?